.env.example

```diff
@@ -0,0 +1,9 @@
+LAB_SESSION_SECRET="cookies-are-tasty"
+LAB_TITLE="OCP4 Workshop"
+LAB_DURATION_HOURS=8h
+LAB_USER_COUNT=50
+LAB_USER_PASS=openshift
+LAB_BLOCKLIST=""
+LAB_USER_PREFIX=evals
+LAB_USER_PAD_ZERO=false
```

Comment:
Controls padding the username with a leading zero when less than 10, e.g `user01`